WalletWasabi.Fluent/Models/Wallets/Address.cs

```diff
@@ -39,8 +39,7 @@ public Address(KeyManager keyManager, HdPubKey hdPubKey)
 
 	public void Hide()
 	{
-		// Code commented out due to https://github.com/zkSNACKs/WalletWasabi/issues/10177
-		// KeyManager.SetKeyState(KeyState.Locked, HdPubKey);
```

Comment:
There was a meeting a long time ago where we were discussing this problem. And if I remember correctly the conclusion was that it is OK to mark external keys `Locked` because it has a different meaning.
So I would just uncomment this line... and in another PR we could introduce a new enum, called `hidden` to avoid future confusion.

Reply:
Regarding the issue https://github.com/zkSNACKs/WalletWasabi/issues/10177... someone said that we cannot really do anything with it.